Modules/xASL_module_ASL.m

```diff
@@ -187,6 +188,22 @@
 	x.Q.NumberEchoTimes = 1;
 end
 
+%% D6. DeltaM parsing - check if all/some volumes are deltams
+% If TSV file exist
+[pathDir, pathASL4D] = fileparts(x.P.Path_ASL4D);
+pathASL4Dcontext = fullfile(pathDir, [pathASL4D '_aslcontext.tsv']);
+% We don't have a subtraction image by default
```

Comment:
Unless there is only a single volume

Reply:
Agree. Will change.

Reply:
On the other hand. A single volume will still have deltam in the context, so we don't need to program this option as this already works. ASLcontext always have to be present and is checked in import.

Reply:
Yes, but then this needs to be automated in the import, always putting 1 volume to `deltaM` with a warning, if `ASLcontext` wasn't provided yet.

Reply:
Agree. Added this warning.